distributed/client.py

```diff
@@ -3033,6 +3034,15 @@ def _get_computation_code(
                 or fr.f_code.co_name in ("<listcomp>", "<dictcomp>")
             ):
                 continue
+
+            # Ignore IPython specific wrapping IPython interactive related functions
+            elif (
+                hasattr(fr.f_back, "f_globals")
+                and sys.modules[fr.f_back.f_globals["__name__"]].__name__  # type: ignore
+                == "IPython.core.interactiveshell"
+            ):
+                if is_ipython_idx is None:
+                    is_ipython_idx = i + 1  # Everything past this index.
```

Comment:
nit: this seems like odd design. it would make more sense to me to simply `break` here, rather than add code to list but then slice list to not include